### DISCUSSION TITLE
Instruction to get score 100 on the mobile

### DISCUSSION THREAD

**Discussion by navisoft** [GoogleChrome/lighthouse#12949] [Ideas]
I have used the lighthouse to monitor a website for the performance. However, it's really hard to get 100 score for the mobile. I only can get the score 100 for the mobile with the site that contains only a static text without css, javascript.
I'm not sure if lighthouse team considers that the website contains only a static text is popular nowaday for the modern website.
Of course, the PWA is not standard today yet and even for the PWA, we must load for "full state" mode as well.
So we might need an instruction for the website development to get store 100 on the mobile, of course the website contains the css, javascript and maybe some static images as well.

**Comment by choutkamartin** [comment]
I would take it with a grain of salt, if the performance score is between 95-100 on desktop, I would say, the website is okay. Getting 100 score on mobile is almost impossible. Don't depend on it way too much.
